web/src/components/changes.js

```diff
@@ -137,6 +138,7 @@ class ChangesTable extends React.Component {
                       <td><a href={addUrlField('authors', x.author)}>{x.author}</a></td>
                       <td>{changeUrl(x, x.title)}</td>
                       {this.props.mergeable ? <td>{x.mergeable}</td> : null}
+                      <td align="center">{ x.changed_files + x.additions + x.deletions}</td>
```

Comment:
Could you extract into a function ?

Also what about a complexity function such as:
m = additions + deletions
score_m = 1 if m < 10
socre_m = 2 if m < 100
...
fc = file changes count
score_fc =  1 if fc < 2
score_fc = 2 if fc < 4
...

score = score_m + score_fc + ...

?

Reply:
I have factorized this code in the reducers to make it more useful globally. Btw I have added a bubble graph in the /merged-changes route